src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -283,6 +285,21 @@
             return this.itemsContainer;
         }
 
+        toggleSourceFocus(isFocused) {
+            const isDisabled = this.container.classList.contains('ibexa-dropdown--disabled');
+            const shouldFocus = isFocused && !isDisabled;
```

Comment:
As I understand, source input is still focusable, so there's possibility that user will focus on it, but as dropdown is disabled, it won't be visible as focused.

Probably source input should be disabled as well, when dropdown is disabled?